AmplifyTests/CategoryTests/Storage/StoragePathTests.swift

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+
+@testable import Amplify
+@testable import AmplifyTestCommon
+
+class StoragePathTests: XCTestCase {
```

Comment:
what does uploading at the root of the S3 bucket look like in terms of how to set the StoragePath?

Reply:
Accessing the root bucket is not supported, there needs to be a sub-folder.

Reply:
It can't be an empty string, so they can still access the resource directly that are at the root.